### DISCUSSION TITLE
MAUI - How to check if Bluetooth is enabled

### DISCUSSION THREAD

**Discussion by rgomez90** [shinyorg/shiny#1074] [Q&A] [answered]
I am migrating a Xamarin Forms app to .NET MAUI and I would want to know how to check if bluetooth is enabled in the device?

I navigated the docs and also the samples but didn't find anything. 

The only thing I found is `IBleDelegate` where adapter state changes can be tracked.

I just want to check at the start of the application if bluetooth is enabled, so I can open the device settings for the user to activate it if disabled.

**Comment by aritchie** [comment, marked as answer]
IBleManager.RequestAccess does everything you need.

Please be respectful.  If the docs are "useless", you're welcome to submit a PR.  Remember this library is free and MAUI/NET6 support is alpha only at the moment.

**Reply by rgomez90** [reply]
@aritchie Thank you, will  take a look at that. 
And sorry 🙏,  it was not meant "bad", but yes it was a bad word choice. I appreciate every second of work you and the contributors put into this.
I edited my question removing it, my apologies again.
